tiny_dnn/core/kernels/conv2d_op_avx.h

```diff
@@ -78,17 +81,17 @@ void avx_conv2d_5x5_kernel(const core::conv_params &params,
         sum3        = _mm_add_ps(tmp3, sum3);
       }
       __m256 sum  = _mm256_add_ps(_mm256_add_ps(sum0, sum1), sum2);
-      __m128 b    = _mm_load_ss(&bias[o]);
+      __m128 b    = _mm_load_ss(&bias_begin[o]);
       __m128 hsum = hsum256_ps(sum);
       b           = madd128_ss(b, y_bias_scale, sum3);
-      _mm_store_ss(&a[o], _mm_add_ss(hsum, b));
+      _mm_store_ss(&a_begin[o], _mm_add_ss(hsum, b));
     }
   } else {
     const serial_size_t nblocks = out.width_ / 4;
     for (serial_size_t o = 0; o < out.depth_; ++o, oidx += out_area) {
-      float *pa = &a[oidx];
+      float *pa = &a_begin[oidx];
       // init to bias value
-      float b = bias[o] * bias_scale;
+      float b = bias.host_at(0, o) * bias_scale;
```

Comment:
If the last dimesion of Tensor is contiguous and a position for parent dimension is fixed, we can get a pointer from `bias` tensor outside of the loop.

It looks like `bias` variable changed from 1D to 2D....but only last dimension is specified.